sdk/search/azure-search-documents/swagger/readme.md

```diff
@@ -121,10 +123,25 @@ directive:
       where: $
       transform: >-
           return $
-          .replace(/(\@HostParam\(\"indexName\"\) String indexName)/g, "$1\, @HeaderParam\(\"accept\"\) String accept")
-          .replace(/(this.client.getIndexName\(\),)/g, "$1 accept,")
+          .replace(/(\@QueryParam\(\"api\-version\"\) String apiVersion)/g, "$1\, @HeaderParam\(\"accept\"\) String accept")
+          .replace(/(this\.client\.getApiVersion\(\)\,)/g, "$1 accept,")
           .replace(/(public Mono\<(.*)\) \{)/g, "$1\n\t\tfinal String accept \= \"application\/json\;odata\.metadata\=none\"\;\n")
 
+    # Change service resource impl to take Accept header parameter
```

Comment:
The autorest.core version we used do not support the change. Update to latest autorest.core one works well, but break other transformers. I think we still need to switch latest autorest.core. I dont think it can deliver this release.

Reply:
I believe this could be added as a transform like

```json
"parameters": [
  {
    "name": "Accept",
    "in": "headers",
    "required": true,
    "type": "string",
    "enum": [
      "application/json;odata.metadata=minimal"
    ]
  }
]
```

Reply:
Does it add to parameters? Or replace to accept?

Reply:
I will have a try. Basically,
parameters.push does not work

Reply:
It did not work. Jianghao has pull the json after swagger.

Somehow autorest.core did not generate the right parameters for modeler to parse. That's the main reason. For .NET, I guess they use latest autorest.core version, which generate the right code for Java too. Unfortunately, other transformers get a mess if using the latest.

Reply:
Can circle back on this in the future since this is internal code, no need to hold up this change which is needed.